src/core/Microsoft.PowerToys.Settings.UI/Views/GeneralPage.xaml

```diff
@@ -13,32 +13,39 @@
-                    <RadioButton Content="Dark"/>
-                    <RadioButton Content="Light"/>
-                    <RadioButton Content="System default" IsChecked="True"/>
+                    <RadioButton x:Name="Rodio_Theme_Dark" Content="Dark" Tag="Dark" Checked="Theme_Changed"/>
+                    <RadioButton x:Name="Rodio_Theme_Light"  Content="Light" Tag="Light" Checked="Theme_Changed"/>
+                    <RadioButton x:Name="Rodio_Theme_Default"  Content="System default" Tag="System" Checked="Theme_Changed"/>
                 </muxc:RadioButtons>
 
+                <!--
                 <ToggleSwitch Header="Disable telemetry" IsOn="True" Margin="0,14,0,0" />
                 <TextBlock Text="PowerToys currently respects the Windows data &amp; feedback setting" Opacity="0.8" Margin="0,0,0,0" />
+                    -->
 
+                <Button Background="{ThemeResource SystemAccentColor}" Content="Restart as admin" Margin="0,10,0,0" Foreground="White" Click="Restart_Elevated" />
+
+                <!--
                 <TextBlock Text="Default apps" Style="{StaticResource SubtitleTextBlockStyle}" Margin="0,34,0,8"/>
                 <ComboBox Header="Shell" SelectedIndex="0" MinWidth="240" Margin="0,14,0,0">
                     <ComboBoxItem>PowerShell</ComboBoxItem>
                 </ComboBox>
                 <ComboBox Header="Terminal" SelectedIndex="0" MinWidth="240" Margin="0,14,0,0">
                     <ComboBoxItem>Windows Console</ComboBoxItem>
                 </ComboBox>
+               -->
+
                 <TextBlock Text="About PowerToys" Style="{StaticResource SubtitleTextBlockStyle}" Margin="0,34,0,8"/>
                 <TextBlock FontWeight="Bold" Text="Version 0.15.1.0" Margin="0,14,0,0" />
-                <Button Background="{ThemeResource SystemAccentColor}" Content="Check for updates" Margin="0,10,0,0" Foreground="White"/>
-                <!-- TO DO: The styling of this button should be improved so the hover/pressed states are representing the SystemAccentColor -->
+                <!--<Button Background="{ThemeResource SystemAccentColor}" Content="Check for updates" Margin="0,10,0,0" Foreground="White" />
+                 TO DO: The styling of this button should be improved so the hover/pressed states are representing the SystemAccentColor -->
 
-                <HyperlinkButton Content="Report a bug" Margin="0,14,0,0" />
-                <HyperlinkButton Content="Request a feature"  />
-                <HyperlinkButton Content="Privacy statement"  />
+                <HyperlinkButton Content="Report a bug" Margin="0,14,0,0" NavigateUri="https://github.com/microsoft/PowerToys/issues" />
+                <HyperlinkButton Content="Request a feature"  NavigateUri="https://github.com/microsoft/PowerToys/issues"/>
+                <HyperlinkButton Content="Privacy statement" NavigateUri="https://privacy.microsoft.com/en-us/privacystatement" />
```

Comment:
lets chance the privacy link to http://go.microsoft.com/fwlink/?LinkId=521839 so it is consistent with the readme.

Reply:
https://github.com/microsoft/PowerToys/pull/1664/commits/033b33be6b9955024de358e010fa53c6bb7debc8